tools/Ecoregionalization_workflow/BRT_model.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="brt_analysis" name="BRT tool prediction" version="0.1.0+galaxy0" >
+    <description>for species distribution modelling</description>
+    <requirements>
+       <requirement type="package" version="4.3.0">r-base</requirement>
+       <requirement type="package" version="1.3_14">r-dismo</requirement>
+       <requirement type="package" version="2.1.8">r-gbm</requirement>
+       <requirement type="package" version="3.4.2">r-ggplot2</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/brt.R'
+         '$enviro'
+         '$species_files'
+         '$input3'
+         '$output1'
+         '$output2'
+         '$output3'
+         '$output4'
+    ]]></command>
+    <inputs>
+      <param name="enviro" type="data" format="txt,csv,tsv" label="Environment Data"/>
+      <param type="data" name="species_files" label="Occurrences Data File" format="tabular,txt,csv" multiple="True" />
+      <param name="input3" type="text" label="write the name of your abiotic parameters (comma separated)"/>
+    </inputs>
+    <outputs>
+      <collection name="output1" type="list" label="Prediction files">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+_brts_pred_ceamarc)\.txt" format="txt"/>
+      </collection>
+      <collection name="output2" type="list" label="Validation files (Taxa, AUC, Tree complexity, Total deviance explained)">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+_brts_validation_ceamarc)\.tsv" format="tabular" />
+      </collection>
+      <collection name="output3" type="list" label="Species distribution prediction maps" >
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" format="png" />
+      </collection>  
+      <collection name="output4" type="list" label="Partial dependence plots" >
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.pdf" format="pdf" />  
+      </collection>
+    </outputs>
+    <tests>
+        <test>
+            <param name="enviro" value="ceamarc_env.csv"/>
+            <param name="species_files" value="cnidaria_filtered.csv"/>
+            <param name="input3" value="Carbo,Grav,Maxbearing,Maxmagnit,Meancurmag,Meansal,Meantheta,Mud,Prof,Rugosity,Sand,Seaice_prod,Sili,Slope,Standcurmag,Standsal,Standtheta"/>
+            <output_collection name='output1' type = "list" count="1"/>
```

Comment:
you could add here asserts as well, that are checking for number of columns or number of lines of the tabular